src/config_resolver.go

```diff
@@ -16,10 +16,11 @@ package main
 import (
 	"bytes"
 	"encoding/json"
-	"github.com/elodina/go-avro"
 	"os"
 	"text/template"
 	"time"
+
+	"github.com/elodina/go-avro"
```

Comment:
Then it must be a change in go-1.8!  VSCode is probably doing the same thing under the hood.